python/GafferUI/CompoundEditor.py

```diff
@@ -453,6 +456,20 @@ def __updateStyles( self ) :
 		self._qtWidget().setProperty( "gafferNumChildren", GafferUI._Variant.toVariant( "%d" % len(self) ) )
 		self._repolish()
 
+	def __configureTab( self, editor ) :
+
+		button = GafferUI.Button( image="deleteSmall.png", hasFrame=False )
+		button._qtWidget().setFixedSize( 11, 11 )
+
+		editor.__removeButton = button
+		editor.__removeButtonConnection = button.clickedSignal().connect( IECore.curry(
```

Comment:
Prefer `functools.partial()` to `IECore.curry()` - it's the standard python way.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/commit/7655968d95872c26a5e8524a295ac44cf5a42f71